Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM golang:1.21-alpine as base
-MAINTAINER team@pganalyze.com
+FROM golang:1.21-alpine AS base
 LABEL org.opencontainers.image.authors="team@pganalyze.com"
 
-ENV GOPATH /go
-ENV HOME_DIR /home/pganalyze
-ENV CODE_DIR $GOPATH/src/github.com/pganalyze/collector
+ENV GOPATH=/go
```

Comment:
For other reviewers, the space-based syntax here is [now deprecated](https://docs.docker.com/reference/dockerfile/#env).